service/r2/kv/store.go

```diff
@@ -37,6 +38,8 @@ type store struct {
 	updateHelper rules.RuleSetUpdateHelper
 }
 
+var errNilValidator = errors.New("no validator set on StoreOptions so validation is not applicable")
```

Comment:
super nit: might read a little better to say "no validator set so validation is not applicable".